.github/workflows/meta-tuist.yml

```diff
@@ -107,3 +140,32 @@ jobs:
       - uses: actions/checkout@v3
       - name: Run
         run: make workspace/lint/lockfiles
+
+  cache-warm:
+    name: Cache warm
+    runs-on: macos-13
+    env:
+      TUIST_INCLUDE_TUIST_CLOUD: 1
+    steps:
+      - uses: actions/checkout@v3
+      - name: Select Xcode
+        run: sudo xcode-select -switch /Applications/Xcode_$(cat .xcode-version).app
+      - uses: actions/cache@v3
+        name: 'Cache Tuist .build folder'
+        with:
+          path: .build
+          key: ${{ runner.os }}-${{ hashFiles('.xcode-version') }}-spm-${{ hashFiles('Package.resolved') }}-git-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-${{ hashFiles('.xcode-version') }}-spm-${{ hashFiles('Package.resolved') }}
+            ${{ runner.os }}-${{ hashFiles('.xcode-version') }}-spm
+      - uses: actions/checkout@v3
+        with:
+          repository: tuist/TuistCloud
+          token: ${{ secrets.TUIST_CLOUD_TOKEN }}
+          path: TuistCloud
+          fetch-depth: 0
```

Comment:
We should be cloning the version the repository is [pinned to](https://github.com/tuist/tuist/blob/main/.tuist-cloud-version)

Reply:
This just clones the repository. We still change the commit in `workspace/up`. The clone in `up.sh` happens the same way.